rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,74 @@
+
+As a consumer of HCA data, I will be more confident in using metadata, raw data, or analysis products from a project if all data and data provenance are available and easily referenced.
+
+As a consumer of HCA data, I do not want to see Integration Test projects on the DataBrowser or DataStore, especially if I have a notification set up for new projects as these will trigger my notification but don’t actually represent new data.
+
+As a member of the DCP, I cannot do a comprehensive final check on data on the DataBrowser or DataStore unless I see it as the data consumer will see it.
+
+As a member of the DCP, I do not want inaccurate metadata or products of incorrect data analysis being shown to the public. 
+Examples of inaccuracies that were/are publicly accessible:
+- Optimus results on Smart-seq2 data ([#2314](https://github.com/HumanCellAtlas/data-store/issues/2314))
+- ‘Homo sapiens’ vs ‘homo sapiens’ ([ZD ticket 158](https://humancellatlas.zendesk.com/agent/tickets/158))
+- Project ingested at the wrong schema version ([#2421](https://github.com/HumanCellAtlas/data-store/issues/2421))
+- Project Metadata tsv download threw an error
+- Contributor name misspelled ("Tom Mithcell")
+
+As a contributor whose name is associated with the project, I want to be allowed time to review the full details of my contribution as for accuracy before it becomes publicly available. I want this review to be through the vantage of a potential user. (possibly 2nd phase)
+
+As a contributor, I am curious where my submission is in the DCP’s data lifecycle after I have provided the metadata spreadsheet all the way until it is released. (possibly 2nd phase)
+
+
+## Functional Specifications
+
+- A mechanism for DCP members to sign-in to the DataBrowser and DataStore.
+- A mechanism for a project to be visible only for a user that is signed-in.
+- Ensure Secondary Analysis and Matrix Service are able to access the data hidden from signed-out users and to contribute their products back to the project
+- A mechanism for Data Operations to switch a project from not visible for signed-out users to visible for all users (data release).
+- A ‘checklist’ and validation procedure for Data Operations to perform on each project within a week of being deemed version-complete.
+- A mechanism for a data contributor to sign-in to the DataBrowser and DataStore, and be given access their own project before it is visible for all users. (possibly 2nd phase)
+- A dashboard on the project page for signed-in users to track where a given project is in the data lifecycle. (possibly 2nd phase)
+
+*The portions of the proposal addressing contributor access to only their projects have been noted as “possibly 2nd phase”. In anticipation of a logical breakpoint in technical effort required, the implementation of DCP-only access should not be delayed by these additional requirements.*
```

Comment:
This is a long least of requirements. I think it would be helpful for the engineering plan to define which business requirements must be serviced by the MVP solutions for this process and which can be delayed in more specific manner than this. I think the phased requirement list in the citation RFC does this well.

Reply:
I have more clearly distinguish the functional specifications between Phase 1 & 2 in ca09319

Reply:
Even more Phase break down in eef5941

Reply:
Resolving this thread, please renew comments if I can address things further in the RFC.